app/assets/styles/app.scss

```diff
@@ -180,7 +180,7 @@ h1.ui.header, h2.ui.header {
 }
 
 // Message
-.ui.positive.message {
+.ui.positive.message, .ui.success.message, .ui.error.message {
```

Comment:
rn the color when you successfully geocode is red (because of the secondary app color) -- could we change that to be the original success message instead? or do you think the green of success would clash w the teal in the app? This is what i mean
<img width="868" alt="screen shot 2017-02-13 at 11 40 10 pm" src="https://cloud.githubusercontent.com/assets/9835059/22915542/0a7e34fe-f246-11e6-86a9-be4aaa3b4eef.png">